zigConsole/.vscode/launch.json

```diff
@@ -0,0 +1,211 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Local AMD64",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/zig-deploy/x86_64-linux-gnu/debug/bin/__change__",
```

Comment:
when using the `build-debug-amd64` it's appears to build it correctly, but then it's create the `${workspaceFolder}/zig-out/bin/apptest` instead. So, changing here appears to work.

This is wat you are expecting, or the task `build-debug-amd64` should be changed to output to a another folder structure?